packages/unlock-js/src/subgraph/schema.graphql

```diff
@@ -22,6 +22,17 @@ query allLocks(
     version
     createdAtBlock
     totalKeys
+    keys {
+      id
+      tokenId
+      owner
+      manager
+      expiration
+      tokenURI
+      createdAtBlock
+      cancelled
+      transactionsHash
+    }
   }
```

Comment:
Would avoid this. This is fetching thousands of keys on every getLock call where you might be getting multiple of the locks.

Refactor parts where you are using getLock to getKeys or create another query function.